src/cmd/build.js

```diff
@@ -37,14 +39,55 @@ export type PackageCreatorParams = {
   artifactsDir: string,
 };
 
+export type LocalizedNameParams = {
+  messageFile: string,
+  manifestData: ExtensionManifest,
+}
+
 export type PackageCreatorFn =
```

Comment:
This used to be defined right above `defaultPackageCreator` so you will need to move it back there. This helps group the type definitions with the functions they relate to.